.circleci/config.yml

```diff
@@ -108,3 +108,19 @@ workflows:
           filters:
             branches:
               only: master  # don't deploy to GitHub pages from other branches
+
+  weekly-webpage:
```

Comment:
doesn't this need to be listed under jobs?

Reply:
It has jobs of its own, so.... how?

Reply:
I suppose it might be simpler to add a scheduler to the existing job. The `doctest` part wouldn't be needed, but that might be better than having so much extra configuration which is largely a duplicate.